lib/Dialect/StandardOps/Ops.cpp

```diff
@@ -155,7 +156,8 @@ static void printStandardCastOp(Operation *op, OpAsmPrinter *p) {
 }
 
 /// A custom cast operation verifier.
-template <typename T> static LogicalResult verifyCastOp(T op) {
+template <typename T>
+static LogicalResult verifyCastOp(T op) {
```

Comment:
our internal clang-format put this on a single line apparently

Reply:
Do I need to react on this? Was just not sure because the issue has already the "ready to pull" label.

Reply:
Sorry I missed your answer, yes this block our integration internally. If you had fixed it then it would have gone through (I could also have cloned and fix myself)

Reply:
I could have fixed it myself by cloning and patching it, but indeed this was blocking the integration (after we mark it "ready to pull", the PRs are going through some tooling to integrate this internally before we push it out to the master branch again)